moped-editor/src/views/projects/projectView/ProjectComponents/KnackComponentAutocomplete.js

```diff
@@ -45,18 +60,18 @@ const SignalComponentAutocomplete = React.forwardRef(
       return <CircularProgress color="primary" size={20} />;
     } else if (error) {
       return (
-        <Alert severity="error">{`Unable to load signal list: ${error}`}</Alert>
+        <Alert severity="error">{`Unable to load ${componentLabel} list: ${error}`}</Alert>
       );
     }
 
     return (
       <Autocomplete
         className={classes}
-        id="signal-id"
+        id="knack-component-autocomplete-id"
```

Comment:
@chiaberry Could you remove the `-id` part of this id string? This is causing the 1PW extension to want to autofill on it for some reason. 🤷 This is an existing issue in prod for this autocomplete signals so not something you introduced.

<img width="594" alt="Screenshot 2024-09-23 at 6 08 01 PM" src="https://github.com/user-attachments/assets/fa34cac8-acc0-4bf8-b6d3-9f5c638603b1">

Reply:
yay! thank you for squeezing this in. Such a nice detail to tie up 🙌